starter-code/index.js

```diff
@@ -1,10 +1,32 @@
-var SortedList = function() {};
+var SortedList = function () {
+  this.items = [];
+  this.length = this.items.length;
```

Comment:
This would work, but better:
```
this.length = 0;
```
You are simulating your own data structure with its own properties.